lib/KTable/index.vue

```diff
@@ -0,0 +1,564 @@
+<template>
+
+  <div ref="tableWrapper" class="k-table-wrapper">
+    <template v-if="dataLoading">
+      <p><KCircularLoader /></p>
+    </template>
+    <template v-else>
+      <table v-if="!isTableEmpty" class="k-table" role="grid">
+        <caption v-if="caption" class="visuallyhidden">
+          {{ caption }}
+        </caption>
+        <thead>
+          <tr ref="stickyHeader">
+            <th
+              v-for="(header, index) in headers"
+              :ref="'header-' + index"
+              :key="index"
+              tabindex="0"
+              :aria-sort="isColumnSortable(index) ? getAriaSort(index) : null"
+              :class="{
+                [$computedClass(coreOutlineFocus)]: true,
+                sortable: isColumnSortable(index),
+                'sticky-header': true,
+                'sticky-column': index === 0,
+              }"
+              :style="[getHeaderStyle(header),
+                       isColumnSortActive(index) ? { color: $themeBrand.primary.v_1000 } : { color: $themePalette.grey.v_800 },
+                       { backgroundColor: $themePalette.white } ,
+                       isColumnFocused(index) ? { backgroundColor: $themePalette.grey.v_50 } : {},
+                       { textAlign: getTextAlign(header.dataType) }]"
+              role="columnheader"
+              data-focus="true"
+              :aria-colindex="index + 1"
+              @click="sortable ? handleSort(index) : null"
+              @keydown="handleKeydown($event, -1, index)"
+            >
+              <slot name="header" :header="header" :colindex="index">
```

Comment:
I think here you intended `colIndex`?

Would be nice to double-check on other places as sometimes it's `colindex` and sometimes `colIndex` (now talking only about the context of scoped slot, for other variables it's fine). As JS is case-sensitive, let's make sure we use one name for the same thing - in examples, documentation, and internal implementation

Reply:
Checked all places, done required changes.